algebra/src/msm/variable_base.rs

```diff
@@ -1,12 +1,90 @@
-use crate::{
-    AffineCurve, BigInteger, Field, FpParameters, PrimeField,
-    ProjectiveCurve,
-};
+use crate::{AffineCurve, BigInteger, Field, FpParameters, PrimeField, ProjectiveCurve};
 use rayon::prelude::*;
 
 pub struct VariableBaseMSM;
 
 impl VariableBaseMSM {
+    pub fn multi_scalar_mul_affine<G: AffineCurve>(
```

Comment:
This function seems to be very similar to msm_affine. Let's bench one versus the other.